src/include/sof/audio/component.h

```diff
@@ -728,6 +728,8 @@ void sys_comp_module_src_interface_init(void);
 void sys_comp_module_tdfb_interface_init(void);
 void sys_comp_module_volume_interface_init(void);
 
+void sys_comp_module_dts_interface_init(void);
```

Comment:
nitpick: this should be in line above, next to the others.

Reply:
not in alphabetical order